lib/common/grpc-service.js

```diff
@@ -266,6 +266,25 @@ GrpcService.prototype.request = function(protoOpts, reqOpts, callback) {
     request: function(_, onResponse) {
       respError = null;
 
+      if (stream) {
```

Comment:
I think we should definitely allow only a callback the same as other similar APIs. It should be pretty easy (oh man, famous last words) to detect an API is a streaming one and just use `concat` if a callback is given.